test/email_alert_api_test.rb

```diff
@@ -79,6 +79,48 @@
 
         assert_equal("weekly", subscription_attrs.fetch("frequency"))
       end
+
+      it "returns the subscription attributes on /latest" do
+        subscription_attrs = api_client.get_latest_matching_subscription(1)
+          .to_hash
+          .fetch("subscription")
+
+        assert_equal(1, subscription_attrs.fetch("id"))
+      end
+    end
+
+    describe "user unsubscribed then resubscribed at the same frequency" do
+      before do
+        stub_email_alert_api_has_subscriptions(
+          [1, "weekly", ended: true],
+          [2, "weekly"]
+        )
+      end
+
+      it "returns the latest subscription attributes on /latest" do
+        subscription_attrs = api_client.get_latest_matching_subscription(1) # notice the 1
```

Comment:
Rather use a comment, could the ids be changed to something more obvious? i.e. a larger number?